scripts/report_scripts/variantreport_p_sample.Rmd

```diff
@@ -289,8 +275,15 @@ p1 <- read.table(file.path(location_sigmuts,"P1_outbreakinfo_mutation_report_dat
 p1$alt_aa <- na_if(p1$alt_aa, '')
 p1$alt_aa[is.na(p1$alt_aa)] <- "-"
 p1$AA_mut <- paste0(p1$ref_aa, p1$codon_num, p1$alt_aa)
-################
+######
+b16172 <- read.table(file.path(location_sigmuts,"B16172_outbreakinfo_mutation_report_data_2021-05-26.tsv"),
```

Comment:
Please add these files as explicit inputs.  Accessing arbitrary files from within reports is not good because Snakemake doesn't know about these file accesses, so it cannot arrange for these files to be available.

(This is a problem when distributing jobs to clusters without a shared file system.)

Reply:
Yes the plan is to change this whole part to be dynamic and everything. But since this is kind of a major restructuring process I pushed that away so far until we have everything else up and running.